styles.css

```diff
@@ -0,0 +1,186 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0px;
+    font-family: sans-serif;
+}
+
+/* MAIN LAYOUT */
+.twitter-grid > * {
+    min-height: 100vh;
+}
+
+.three-columns {
+    display: grid;
+    grid-template-columns: 1fr 3fr 1fr;
+}
+
+/* LEFT SIDE */
+
+.two-columns {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    gap: 20px;
+}
+
+.align-center {
+    align-items: center;
+}
+
+.left-hand-menu {
+    display: grid;
+    grid-template-rows: 80px 2fr 80px;
+    gap: 10px;
+}
+
+.left-box {
+    margin: 10px;
+}
+
+.twitter-icon {
+    padding-left: 45px;
+}
+
+.picture-sm {
+    height: 50px;
+    width: 50px;
+}
+ 
+.text {
+    color: grey;
+    font-size: 20px;
+}
+
+.text-name {
+    color: grey;
+    font-size: 20px;
+}
+
+h2 {
+    padding-left: 15px;
+}
+
+h3, li:hover {
+    opacity: 0.6;
+}
+
+/* TWEET LAYOUT */
+
+.tweet-feed {
+    display: grid;
+    grid-template-rows: auto;
+}
+
+.seven-columns {
+    display: grid;
+    grid-template-columns: repeat(5, 32px) 1fr auto;
+    gap: 20px;
+}
+
+.middle-box {
+    margin: 3px;
+    border: 2px solid lightgray;
+}
+
+.post {
+    margin: 10px;
+}
+
+.bear-image, 
+.inter-image,
+.khaled-image {
+    height: 50px;
+    width: 50px;
+}
+
+.bear-image, 
+.khaled-image {
+    border-radius: 50%;
+}
+
+.reaction-section > * {
+    padding: 2px;
+    margin: 2px;
+    width: 24px;
+    height: 24px;
+}
+
+.reaction-section {
+    list-style-type: none;
+}
+
+.grid-auto-column {
+    display: grid;
+    grid-auto-flow: column;
+}
+
+.book-cover {
+    margin-left: 10px;
+}
+
+button:hover {
```

Comment:
never thought to make the button hover as well thank you for reminding me to do this. Great way to make your website look more pretty.